src/data/finset/lattice.lean

```diff
@@ -677,6 +677,16 @@ lemma sup_finset_image {f : γ → α} {g : α → β} {s : finset γ} :
   s.sup (g ∘ f) = (s.image f).sup g :=
 by { simp_rw [sup_eq_supr, comp_app], rw supr_finset_image, }
 
+lemma sup_finset_image' {α : Type*} {β : Type*} {γ : Type*}
```

Comment:
I think we should delete the original `sup_finset_image`, use this instead, and move it into the `sup` section earlier in the file.

Reply:
The other one does not require `[decidable_eq γ]`.

Reply:
Ah, the linter catches anyway that that is an unnecessary constraint. I'll test your suggestion.